### ISSUE TITLE
NOTICE file mistakes with Third-Party Content

### ISSUE THREAD

**Issue by joakime** [eclipse-ee4j/tyrus#645]
The eclipse-ee4j/websocket-api has erroneous IPzilla entries for the for the following dependencies.

* Google Guava Version: 18.0 (PB Orbit CQ12184) - https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16060
* atinject (Package javax.inject) Version: 1.0 (PB Orbit CQ3578) - https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16061
* javax.validation:validation-api:jar:1.1.0.Final (PB CQ15114 Type A) - https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16062

Can you confirm yes or no that these 3 dependencies should be reassigned from websocket-api to tyrus?

This was discovered by Mark Thomas during a PR review of the generated NOTICE file for websocket-api.
See: https://github.com/eclipse-ee4j/websocket-api/pull/271

A discussion on ee4j-build@eclipse.org was started
https://dev.eclipse.org/mhonarc/lists/ee4j-build/msg00195.html

A possible resolution was proposed by @waynebeaton and we want to make sure that these 3 dependencies are supposed to belong to the Tyrus project.
https://dev.eclipse.org/mhonarc/lists/ee4j-build/msg00199.html

**Comment by bravehorsie** [comment]
Yes, indeed, all those dependencies belong to Tyrus and were filed to websocket-api by mistake.  It would be great if solution proposed by @waynebeaton can be done.

**Comment by keilw** [comment]
Was Web Socket/Tyrus unchanged in Java EE 8? Otherwise the `javax.validation:validation-api:jar:1.1.0.Final` dependency is a little hard to understand, since Java EE 8 contains Bean Validation 2.0 already.

**Comment by joakime** [comment]
Just to keep everyone in the loop.

An Eclipse IP issue has been filed to move these dependencies from websocket-api to tyrus.
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=17668

@m0mus can you nudge the correct people to get this pushed through?  It would help websocket-api + tyrus with their Oct 1st deliverables (as tyrus needs to depend on websocket-api's new jakarta maven coordinates)

**Comment by joakime** [comment]
I've bumped the ipzilla issue, hopefully this get addressed soon.

**Comment by apupier** [comment]
The ipzilla issue is marked as resolved https://dev.eclipse.org/ipzilla/show_bug.cgi?id=17668

we can notice that the component is now ee4j.tyrus for the 3 CQs:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16060
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16061
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16062
